ssg/contributors.py

```diff
@@ -42,6 +44,8 @@
     # Joseph D Lenox
     "jdlenox@labnet.rockwellcollins.lab": "Joseph.Lenox@collins.com",
     "lordofhyphens@gmail.com": "Joseph.Lenox@collins.com",
+    "Joseph.Lenox@corp.rockwellcollins.com": "Joseph.Lenox@collins.com",
+    "joseph.lenox@collins.com": "Joseph.Lenox@collins.com",
```

Comment:
@lenox-joseph Hi, do you have a preferred name and email to put in the contributors list?

Reply:
@yuumasato
Joseph Lenox
joseph.lenox@collins.com

Reply:
Updated.

Reply:
> Updated.

Thank you. Sorry about having the pile of associated email addresses.